### ISSUE TITLE
looksLike and related API

### ISSUE THREAD

**Issue by RickEyre** [EventMobi/angular-easy-test#2]
I've noticed a lot of individuals preferring passing the properties as arrays, i.e.

```js
EasyTest.testService('myService', {
  'function': [ 'one', 'two', 'three' ]
};
```
As opposed to a space delimited string. Perhaps we could support both styles.

**Comment by scottybarr** [comment]
:+1: I am one of those people! :p

**Comment by RickEyre** [comment]
@scottybarr heh, should be coming out soon-ish.. this weekend??

**Comment by elliottsj** [comment]
Closed via #17
